### PR TITLE
chore(Doronime): update domain

### PR DESCRIPTION
**Pull request by noxzym**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
 
</details>

### CODE REVIEW

**Comment by DarkVillager** [comment]
> image image

Change it to
`


`

File: websites/D/Doronime/presence.ts [hunk 1]
```diff
@@ -8,96 +8,119 @@ const presence = new Presence({
...
+			if (data.length) {
+				const title = document
+					.querySelector(".Content__title")
+					.textContent.replace(/\[Batch\]|Subtitle Indonesia/g, "")
+					.split("Episode")[0]
+					.trim();
+				presenceData.details =
+					document.location.pathname.slice(1).split("/").length === 2
+						? "Viewing Anime Info"
+						: title;
```
Comment (DarkVillager) [inline websites/D/Doronime/presence.ts:38]:
```suggestion
				presenceData.details = trim(document.querySelector('[aria-current="page"]')?.previousElementSibling?.textContent ?? document.querySelector('h5[class="Content__title"]')?.textContent?.replace(/(Episode [0-9]+)|(Subtitle Indonesia)/gm, ""))
```

File: websites/D/Doronime/presence.ts [hunk 2]
```diff
@@ -8,96 +8,119 @@ const presence = new Presence({
 	};
 
 presence.on("UpdateData", async () => {
+	const capitalize = (str: string[]) =>
+		str.map(x => x[0].toUpperCase() + x.slice(1)).join(" ");
```
Comment (DarkVillager) [inline websites/D/Doronime/presence.ts:12]: Let's also add a way to trim e.g.
```suggestion
	const capitalize = (str: string[]) =>
		str.map(x => x[0].toUpperCase() + x.slice(1)).join(" "),
		trim = (str: string[]) =>
		str.trim();
```
image

File: websites/D/Doronime/presence.ts [hunk 3]
```diff
@@ -8,96 +8,119 @@ const presence = new Presence({
...
+					title.split(" ").length > 1 ? title.split(" ")[1] : title;
+				presenceData.state = artist;
+			} else presenceData.details = "Viewing Anime OST List";
+			break;
+		case "genre":
+			if (data.length) {
+				presenceData.details = "Viewing Anime Genre";
+				presenceData.state = document.querySelector(
+					".Content__tabs > div > span"
+				).textContent;
```
Comment (DarkVillager) [inline websites/D/Doronime/presence.ts:74]: image
```suggestion
				presenceData.state = trim(document.querySelector(
					'[aria-current="page"]'
				)?:.textContent);
```

File: websites/D/Doronime/presence.ts [hunk 4]
```diff
@@ -8,96 +8,119 @@ const presence = new Presence({
...
+			} else presenceData.details = "Viewing Anime Genre List";
+			break;
+		case "season":
+			if (data.length) {
+				presenceData.details = "Viewing Anime Season";
+				presenceData.state = document.querySelector(
+					".Content__tabs > div > span"
+				).textContent;
```
Comment (DarkVillager) [inline websites/D/Doronime/presence.ts:82]: Ditto
```suggestion
				presenceData.state =  trim(document.querySelector('[aria-current="page"]')?.textContent)
```

File: websites/D/Doronime/presence.ts [hunk 5]
```diff
@@ -8,96 +8,119 @@ const presence = new Presence({
...
+			} else presenceData.details = "Viewing Anime Season List";
+			break;
+		case "artist":
+			if (data.length) {
+				presenceData.details = "Viewing OST Artist";
+				presenceData.state = document.querySelector(
+					".Content__tabs > div > span"
+				).textContent;
```
Comment (DarkVillager) [inline websites/D/Doronime/presence.ts:90]: Ditto
```suggestion
				presenceData.state =  trim(document.querySelector('[aria-current="page"]')?.textContent)
```

File: websites/D/Doronime/presence.ts [hunk 6]
```diff
@@ -8,96 +8,119 @@ const presence = new Presence({
...
+			} else presenceData.details = "Viewing Artist List";
+			break;
+		case "producer":
+			if (data.length) {
+				presenceData.details = "Viewing Anime Producer";
+				presenceData.state = document.querySelector(
+					".Content__tabs > div > span"
+				).textContent;
```
Comment (DarkVillager) [inline websites/D/Doronime/presence.ts:98]: Ditto
```suggestion
				presenceData.state =  trim(document.querySelector('[aria-current="page"]')?.textContent)
```

File: websites/D/Doronime/presence.ts [hunk 7]
```diff
@@ -8,96 +8,119 @@ const presence = new Presence({
...
+			} else presenceData.details = "Viewing Anime Producer List";
+			break;
+		case "studio":
+			if (data.length) {
+				presenceData.details = "Viewing Anime Studio";
+				presenceData.state = document.querySelector(
+					".Content__tabs > div > span"
+				).textContent;
```
Comment (DarkVillager) [inline websites/D/Doronime/presence.ts:106]: Ditto
```suggestion
				presenceData.state =  trim(document.querySelector('[aria-current="page"]')?.textContent)
```

File: websites/D/Doronime/presence.ts [hunk 8]
```diff
@@ -8,96 +8,119 @@ const presence = new Presence({
...
+	presenceData.buttons = [
+		{
+			label: data.length
+				? base === "anime"
+					? "View Anime"
+					: base === "ost"
+					? "View OST"
+					: "Visit Doronime"
+				: "Visit Doronime",
+			url: document.location.href,
```
Comment (DarkVillager) [inline websites/D/Doronime/presence.ts:122]:
```suggestion
			url: href,
```

File: websites/D/Doronime/metadata.json [hunk 9]
```diff
@@ -1,16 +1,16 @@
...
 	},
 	"service": "Doronime",
 	"description": {
 		"en": "Download anime, anime music, anime ongoing, or anime batch with Indonesian subtitle",
 		"id": "Unduh anime, anime musik, anime ongoing, atau anime batch dengan subtitle Indonesia"
 	},
-	"url": "doronime.id",
-	"version": "1.0.7",
+	"url": "doroni.me",
+	"version": "1.0.8",
```
Comment (DarkVillager) [inline websites/D/Doronime/metadata.json:13]: This is more than a fix. So update it to:
```suggestion
	"version": "1.1.0",
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**